File: main/simple-array-sum/simple-array-sum.cs
using System;
using System.Linq;

internal static class Solution {
    private static void Main()
    {
        var n = int.Parse(Console.ReadLine());
        var s = Console.ReadLine().Split(' ').Select(int.Parse).Take(n).Sum();
        Console.WriteLine(s);
    }
}

File: main/simple-array-sum/simple-array-sum.cpp
#include <iostream>

int main()
{
    auto sum = 0, n = 0;
    for (std::cin >> n; n > 0; --n) {
        auto k = 0;
        std::cin >> k;
        sum += k;
    }

    std::cout << sum << '\n';
}
